components/suggestion/switch_channel_provider.jsx

```diff
@@ -584,16 +596,44 @@ export default class SwitchChannelProvider extends Provider {
         };
     }
 
-    formatUnreadChannelsAndDispatch(resultsCallback) {
-        const getChannel = makeGetChannel();
+    fetchAndFormatRecentChannels(resultsCallback) {
+        // const getChannel = makeGetChannel();
```

Comment:
```suggestion
```

Reply:
@Willyfrog should I remove it, or wait for review

Reply:
Done, @Willyfrog thanks for the suggestion will take care of things like this next time.

Reply:
no worries, this happens to all of us from time to time :)